src/odemis/driver/tlaptmf.py

```diff
@@ -416,11 +419,11 @@ def GetInfo(self):
         sn, modl, typ, fmv, notes, hwv, state, nc = values
 
         # remove trailing 0's
-        modl = modl.rstrip("\x00")
-        notes = notes.rstrip("\x00")
+        modl = modl.rstrip(b"\x00")
+        notes = notes.rstrip(b"\x00")
 
         # Convert firmware version to a string
-        fmvs = "%d.%d.%d" % ((fmv & 0xff0000) >> 16,
+        fmvs = b"%d.%d.%d" % ((fmv & 0xff0000) >> 16,
```

Comment:
No, please convert to a *str*, as it'll be shown to the user. 
You need to also make sure that the other str are *str* (and not bytes, on Python 3). That's mostly important for the `modl`, as it's used after.